File: .sdk/tm/ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { ShortcutSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await ShortcutSDK.test()
    equal(null !== testsdk, true)
  })

})

File: rb/test/exists_test.rb
# Shortcut SDK exists test

require "minitest/autorun"
require_relative "../Shortcut_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = ShortcutSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
